lib/commands/context.js

```diff
@@ -360,7 +360,7 @@ extensions.getLatestWebviewContextForTitle = async function (regExp) {
   let contexts = await this.getContextsAndViews();
   let matchingCtx;
   for (let ctx of contexts) {
-    if (ctx.view && ((ctx.view.title || '').match(regExp) || (ctx.view.url || '').match(regExp))) {
+    if (ctx.view && ((ctx.view.title && ctx.view.title.match(regExp)) || (ctx.view.url && ctx.view.url.match(regExp)))) {
```

Comment:
no chance that `ctx.view.title` could be anything other than a string?

Reply:
No. We build that in `appium-remote-debugger`. It should either be a string or undefined/null.

Reply:
if it's undefined/null, we can't call `.match` on it, right? or do you mean that either `ctx.view.title` will be a string, or `ctx.view` will be undefined/null

Reply:
One or both of them can be a string, or undefined/null. We only want to check on what it is if it is a string, rather than coercing it to be a string and then checking no matter what (which is what we were doing).

Reply:
I'm sorry, my fault here: I missed that we were indeed checking for truthiness of `ctx.view.title`. all is well